templates/CRM/Case/Form/ActivityToCase.tpl

```diff
@@ -44,7 +44,7 @@
     }
 
     var dataUrl = {/literal}"{crmURL p='civicrm/case/addToCase' q='reset=1' h=0}"{literal};
-    dataUrl += '&activityId=' + activityID + '&caseId=' + currentCaseId + '&cid=' + {/literal}"{$contactID}"{literal} + '&fileOnCaseAction=' + action;
+    dataUrl += '&activityId=' + activityID + '&caseId=' + currentCaseId + '&cid=' + {/literal}"{if !empty($contactID)}{$contactID}{/if}'"{literal} + '&fileOnCaseAction=' + action;
```

Comment:
I don't think this `cid=` feature works the same way it used to. Maybe when the case dropdown was changed to an entityref? It's supposed to make it default the case dropdown to one of the contact's cases. It doesn't seem to do anything currently whether it's set or not.

But it doesn't change the PR - sometimes it is set and sometimes (legitimately) not.

Reply:
@demeritcowboy yeah - I had a feeling this change didn't get made before because of some complexity & the hope for a better fix

Reply:
It was bugging me so I looked closer. It's two things: An extra apostrophe got added here, and also in the typical situation this is populated, it would only have a default case if the contact has multiple cases.

Reply:
that's a good thing